src/sentry/integrations/slack/message_builder/notifications/daily_summary.py

```diff
@@ -39,7 +35,9 @@ def __init__(
         self.recipient = recipient
 
     def linkify_error_title(self, group):
-        link = get_title_link(group, None, False, False, self.notification, ExternalProviders.SLACK)
```

Comment:
I pulled out [the code from the path it was falling into](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/message_builder.py#L130-L133) instead of trying to rewrite `get_title_link` since it's a one line simple thing